updated April history (#314)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/introduction/whats-new.adoc

// .Other updates
// * TODO

[discrete]
[[april-2023]]
== April 2023

.Glossary entries
* xref:apache-web-server[Apache web server]: Updated to remove extraneous IdM definitions

> * xref:certificate-authority[certificate authority]: Updated to remove extraneous IdM definitions

Reviewer: ```suggestion
* xref:certificate-authority[certificate authority]: Renamed from "certificate authorities", and updated to remove extraneous IdM definitions
```

* xref:domain-controller[domain controller]: Updated to remove extraneous IdM definitions
* xref:kerberos-protocol[Kerberos protocol]: Updated to remove extraneous IdM definitions
* xref:kerberos-realm[Kerberos realm]: Updated to remove extraneous IdM definitions
* xref:posix-attributes[POSIX attributes]: Updated to remove extraneous IdM definitions
* xref:web-server[web server]: Updated to remove extraneous IdM definitions

.Other updates

> * Added xref:non-breaking-spaces[non-breaking spaces] between "Red" and "Hat" within the guide.

Reviewer: ```suggestion
* Added a xref:non-breaking-spaces[non-breaking space] between "Red" and "Hat" in each occurrence within the guide.
```

> * xref:shortdesc[Short descriptions]: Added guidance on writing short descriptions and abstracts.

Reviewer: ```suggestion
* xref:shortdesc[Short descriptions]: Added guidance on writing _short descriptions_ (also known as _abstracts_).
```
